docs/Reference.md

```diff
@@ -70,7 +70,7 @@ title: "Reference"
 * [`<ImageField>`](./ImageField.md)
 * [`<ImageInput>`](./ImageInput.md)
 * [`<ImageInputPreview>`](./ImageInput.md#imageinput)
-* [`<Labeled>`](./Inputs.md#using-labeled)
```

Comment:
In `Fields.md` there is one more reference to `Inputs.md#using-labeled` that could be changed for `Labeled.md`

Reply:
Can't find any

Reply:
Right, you removed that part in your `Fields.md`